src/linter.ts

```diff
@@ -42,7 +41,7 @@ export default class Linter {
     }
 
     let currentFile = this.codeDocument.uri.fsPath;
-    let currentDirectory = path.dirname(currentFile);
+    let wsRoot = vscode.workspace.workspaceFolders[0].uri.path

```

Comment:
Hello, according to https://code.visualstudio.com/api/references/vscode-api#workspace this solution won't work in the following cases:

- When linting a file with no workspace in vscode (vscode.workspace.workspaceFolders will be undefined).
- When working with [multiroot-workspaces](https://code.visualstudio.com/docs/editor/workspaces#_multiroot-workspaces) and linting within a folder like `vscode.workspace.workspaceFolders[1]`.

Reply:
I tried to handle this situation in a new comit. I have never written javascript before and was not familiar with vscode extensions :)